test-clients/src/main/java/com/hedera/services/bdd/suites/perf/TokenTransfersLoadProvider.java

```diff
@@ -221,8 +231,9 @@ private HapiApiSpec runTokenTransfers() {
 						}
 					}
 					op = cryptoTransfer(xfers)
-							.hasKnownStatusFrom(NOISY_ALLOWED_STATUSES)
+							.hasKnownStatusFrom(OK, DUPLICATE_TRANSACTION, SUCCESS, UNKNOWN, INSUFFICIENT_PAYER_BALANCE)
 							.hasRetryPrecheckFrom(NOISY_RETRY_PRECHECKS)
+							.hasPrecheckFrom(OK, PLATFORM_NOT_ACTIVE)
```

Comment:
Why is PLATFORM_NOT_ACTIVE acceptable ?

Reply:
If the platform is still not in `ACTIVE` state, the client will get the this value as response code.

Reply:
This is the way EET framework communicates to test application what happened to this request. It's part of error processing already. For the JRS test design, when the test client sends the request to server, normally, the server should be already in `ACTIVE` state. However, for whatever reason, occasionally the server may get delayed and may get into the `ACTIVE` state later and process future requests. We'll still consider the overall test legitimate.

Reply:
A long approach would be a change to JRS, only launches client after the swirlds.log shows the platform is in ACTIVE mode, so we can avoid the timing issue between launching server and launching client

Reply:
That will be a better solution. But this check is still necessary in cases where network glitches may happen.